lib/api/_loaders/element-global.js

```diff
@@ -156,6 +156,12 @@ class ElementGlobal {
     if (locator instanceof Element) {
       this.locator = locator;
     } else {
+      if (typeof locator === 'string' && this.client && this.client.locateStrategy){
+        locator = {
```

Comment:
use a different `const` here instead of overwriting the argument

Reply:
fixed in [5ef16e9](https://github.com/nightwatchjs/nightwatch/pull/3147/commits/5ef16e99353da97df178ca2463588bcdd60f0438)